robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py

```diff
@@ -323,42 +323,36 @@ def _resolve(self):
         assert all(isinstance(v, FrameStamped) for v in vectors_of_interest)
 
         open_POIs = filter(self.is_poi_occupied, vectors_of_interest)
+        open_POIs.sort(key=self.distance_to_poi_area_heuristic)
 
-        return self.select_best_feasible_poi(open_POIs)
+        for poi in open_POIs:
+            if self.distance_to_poi_area(poi):
+                selection = self.create_selection_marker(poi)
```

Comment:
Is deze poi een "selected_pose" zoals de create_selection_marker def nodig heeft?

Reply:
yes